github/tests/__main__.py

```diff
@@ -44,6 +44,10 @@ def main(argv):
         github.tests.Framework.activateTokenAuthMode()
         argv = [arg for arg in argv if arg != "--auth_with_token"]
 
+    if "--auth_with_jwt" in argv:
```

Comment:
One final touch, this could be documented here in the contribution guide:
https://github.com/PyGithub/PyGithub/blob/master/CONTRIBUTING.md#automated-tests